bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -307,7 +307,7 @@ spec:
         - label:
             app.kubernetes.io/managed-by: olm
             app.kubernetes.io/name: grafana-operator
-          name: grafana-operator-controller-manager
+          name: grafana-operator-controller-manager-v5
```

Comment:
If anyone has a better idea for a name, please let me know.

Reply:
I'm fine with this, Unless we make another breaking change to the csv, but then we can just drop the `-v5` or replace it with something else again.

Reply:
Does this have any impact on the kustomize stuff? I can't remember how we build that deployment.

Reply:
I took a look but couldn't find any linking, so I don't think we will create any issues with kustomize.
I took the freedom to remove the 1.33 changes, and instead 1.32 as it currently says in the Makefile